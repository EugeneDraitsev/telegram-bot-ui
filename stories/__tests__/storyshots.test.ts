// eslint-disable-next-line import/no-extraneous-dependencies
// import initStoryshots from '@storybook/addon-storyshots'
//
// initStoryshots({ /* configuration options */ })

describe('Placeholder', () => {
  test.todo('Re-enable initStoryshots when React 18 will be supported')
})
