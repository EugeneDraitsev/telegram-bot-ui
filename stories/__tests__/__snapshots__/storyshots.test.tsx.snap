// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Buttons Buttons 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <button
    className="sc-9d00118e-0 bKzPjt"
    style={
      {
        "marginRight": 10,
      }
    }
  >
    Default
  </button>
  <button
    className="sc-9d00118e-0 iUVMjA"
    disabled={true}
    style={
      {
        "marginRight": 10,
      }
    }
  >
    Disabled
  </button>
  <button
    className="sc-9d00118e-0 OayCX"
    style={
      {
        "marginRight": 10,
      }
    }
  >
    Loading
  </button>
</div>
`;

exports[`Storyshots Chat Info Chat Info 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="flex justify-center w-full bg-neutral-300 undefined"
  >
    <div
      className="flex items-center w-full max-w-[1200px] text-left py-1 px-4"
    >
      <div
        className="w-[70px] h-[70px] bg-contain rounded-full border-2 border-white"
        style={
          {
            "backgroundImage": "url(/favicon.png)",
          }
        }
      />
      <div
        className="ml-5"
      >
        <div
          className="text-2xl font-light leading-none"
        >
          Kabold —Åamp
        </div>
        <div
          className="text-sm"
        >
          The kaboldest camp
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Chat Info Edge Case Chat Info 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="flex justify-center w-full bg-neutral-300 undefined"
  >
    <div
      className="flex items-center w-full max-w-[1200px] text-left py-1 px-4"
    >
      <div
        className="w-[70px] h-[70px] bg-contain rounded-full border-2 border-white"
        style={
          {
            "backgroundImage": "url(/favicon.png)",
          }
        }
      />
      <div
        className="ml-5"
      >
        <div
          className="text-2xl font-light leading-none"
        >
          Unknown Chat
        </div>
        <div
          className="text-sm"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Graphs Bar Chart 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="sc-4fd34fbd-0 fNCuMC"
  >
    <div
      className="recharts-responsive-container"
      style={
        {
          "height": 400,
          "maxHeight": undefined,
          "minHeight": undefined,
          "minWidth": 0,
          "width": "100%",
        }
      }
    />
    <div
      className="sc-4fd34fbd-1 bGmAMu"
    >
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 ivILxB"
          color="#1f77b4"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user1  (62)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 furick"
          color="#ff7f0e"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user2  (52)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 dltUwp"
          color="#2ca02c"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user3  (30)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 fQokOl"
          color="#d62728"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user4  (27)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 jGtajO"
          color="#9467bd"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user5  (16)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 fSdQEa"
          color="#8c564b"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user6  (10)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 jSmEXD"
          color="#e377c2"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user7  (9)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 bXYYF"
          color="#7f7f7f"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user8  (5)
        </div>
      </div>
      <div
        className="sc-4fd34fbd-2 jYdfXU"
      >
        <div
          className="sc-4fd34fbd-3 iRFyQj"
          color="#bcbd22"
        />
        <div
          className="sc-4fd34fbd-4 bBVCef"
        >
          user9  (3)
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Graphs Pie Chart 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="sc-8a4bcb7a-0 iNbSCs"
  >
    <div
      className="recharts-responsive-container"
      style={
        {
          "height": "100%",
          "maxHeight": undefined,
          "minHeight": undefined,
          "minWidth": 0,
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots Input Icon Input 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="flex"
    style={
      {
        "gap": 10,
      }
    }
  >
    <div>
      <div
        className="sc-6bbadba6-0 fTylqh"
      >
        <input
          className="sc-6bbadba6-2 ltigW"
          defaultValue="Simple input"
        />
        <div
          className="sc-6bbadba6-1 bKJnmb"
        >
          <svg
            fill="none"
            height={24}
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="2"
            viewBox="0 0 24 24"
            width={24}
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
            />
          </svg>
        </div>
      </div>
    </div>
    <div>
      <div
        className="sc-6bbadba6-0 fTylqh"
      >
        <input
          className="sc-6bbadba6-2 ltigW"
          value="Emoji ü¶Ñ"
        />
        <div
          className="sc-6bbadba6-1 bKJnmb"
        >
          üêà
        </div>
      </div>
    </div>
    <div>
      <div
        className="sc-6bbadba6-0 fTylqh"
      >
        <input
          className="sc-6bbadba6-2 iPIqiS"
          style={
            {
              "width": 215,
            }
          }
          value="123,43"
        />
        <div
          className="sc-6bbadba6-1 bKJnmb"
        >
          $
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Input Simple 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="sc-6bbadba6-0 fTylqh"
  >
    <input
      className="sc-6bbadba6-2 hxYXJQ"
      value="Simple input"
    />
  </div>
</div>
`;

exports[`Storyshots Spinners Different Spinners 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <svg
    aria-label="spinner"
    className="sc-c7059ab5-0 eJDjbO"
    style={
      {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-c7059ab5-1 kqgnh"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
  <svg
    aria-label="spinner"
    className="sc-c7059ab5-0 eJDjbO"
    style={
      {
        "height": 105,
        "width": 105,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-c7059ab5-1 kqgnh"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
  <svg
    aria-label="spinner"
    className="sc-c7059ab5-0 eJDjbO"
    style={
      {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-c7059ab5-1 jTPfia"
      color="red"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
  <svg
    aria-label="spinner"
    className="sc-c7059ab5-0 eJDjbO"
    style={
      {
        "height": 100,
        "width": 100,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-c7059ab5-1 fxjpiO"
      color="green"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
</div>
`;

exports[`Storyshots Spinners Simple Spinner 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <svg
    aria-label="spinner"
    className="sc-c7059ab5-0 eJDjbO"
    style={
      {
        "height": 50,
        "width": 50,
      }
    }
    viewBox="25 25 50 50"
  >
    <circle
      className="sc-c7059ab5-1 kqgnh"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth={2}
    />
  </svg>
</div>
`;

exports[`Storyshots Tabs Long Tabs 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="sc-cc4f5331-0 kSmzgg"
  >
    <div
      className="sc-edca5a4e-0 fllopP"
    >
      <div
        className="sc-edca5a4e-1 kCoayL"
      >
        <div
          className="sc-edca5a4e-2 SpdGT tab active"
          onClick={[Function]}
        >
          Tab1
        </div>
        <div
          className="sc-edca5a4e-2 SpdGT tab"
          onClick={[Function]}
        >
          Veeeeeery Loooooong Name Tab
        </div>
        <div
          className="sc-edca5a4e-2 SpdGT tab"
          onClick={[Function]}
        >
          Some other tab
        </div>
        <div
          className="sc-edca5a4e-2 SpdGT tab"
          onClick={[Function]}
        >
          tab 4
        </div>
        <div
          className="sc-edca5a4e-3 dRwWEz"
          tabIndex={0}
          width={0}
          x={0}
        />
      </div>
    </div>
    <div
      className="sc-cc4f5331-1 bVIVTB"
    >
      selected 
      Tab1
       tab!
    </div>
  </div>
</div>
`;

exports[`Storyshots Tabs Simple Tabs 1`] = `
<div
  className="sc-2bff786d-0 cbqWch"
>
  <div
    className="sc-edca5a4e-0 fllopP"
  >
    <div
      className="sc-edca5a4e-1 kCoayL"
    >
      <div
        className="sc-edca5a4e-2 izKQWD tab"
        onClick={[Function]}
      >
        Tab1
      </div>
      <div
        className="sc-edca5a4e-2 izKQWD tab"
        onClick={[Function]}
      >
        Tab 2
      </div>
      <div
        className="sc-edca5a4e-2 izKQWD tab"
        onClick={[Function]}
      >
        Tab 3
      </div>
      <div
        className="sc-edca5a4e-3 jsxhFw"
        tabIndex={-1}
        width={0}
        x={0}
      />
    </div>
  </div>
</div>
`;
